proto/viam/app/data/v1/data.proto

```diff
@@ -27,6 +27,18 @@ service DataService {
 
   // DeleteBinaryDataByIDs deletes binary data based on given IDs.
   rpc DeleteBinaryDataByIDs(DeleteBinaryDataByIDsRequest) returns (DeleteBinaryDataByIDsResponse);
+
+  // AddAnnotationsToBinaryDataByFileID adds annotations to binary data based on given IDs.
+  rpc AddAnnotationsToBinaryDataByFileIDs(AddAnnotationsToBinaryDataByFileIDsRequest) returns (AddAnnotationsToBinaryDataByFileIDsResponse);
+
+  // AddAnnotationsToBinaryDataByFileID adds annotations to binary data based on given filters.
```

Comment:
supernit: maybe "based on the given filter" here and below, just to match the method name

Reply:
done!